docs/getting-started/README.md

```diff
@@ -0,0 +1,151 @@
+
+<!-- TODO: update example to use ecocredit module once updated to v1.1 -->
+
+## Prerequisites
+
+In order to install the `regen` binary, you'll need the following: 
+
+- Git `>=2`
+- Make `>=4`
+- Go `>=1.15`
+
+For more information, see [Prerequisites](./prerequisites). 
+
+## Install Regen
+
+The `regen` binary serves as the node client and the application client. In other words, the `regen` binary can be used to both run a node and interact with it.
+
+Clone the `regen-ledger` repository:
+```
+git clone https://github.com/regen-network/regen-ledger
+```
+
+Change to the `regen-ledger` directory:
+```
+cd regen-ledger
+```
+
+Check out the latest stable version:
+```
+git checkout v1.0.0
+```
+
+Install the `regen` binary:
+```
+make install
+```
+
+Check to make sure the install was successful:
+```
+regen version
+```
+
+You should see `v1.0.0` printed to the console. Now that you have successfully installed the `regen` binary, the next step will be to add a couple test accounts.
+
+## Create Accounts
+
+In this section, you will create two test accounts. You will name the first account `validator` and the second account `delegator`. You will create both accounts using the `test` backend, meaning both accounts will not be securely stored and should not be used in a production environment. When using the `test` backend, accounts are stored in the home directory (more on this in the next section). 
+
+Create `validator` account:
+```
+regen keys add validator --keyring-backend test
+```
+
+Create `delegator` account:
+```
+regen keys add delegator --keyring-backend test
+```
+
+After running each command, information about each account will be printed to the console. The next step will be to initialize the node.
+
+## Initialize Node
+
+Initializing the node will create the `config` and `data` directories within the home directory. The `config` directory is where configuration files for the node are stored and the `data` directory is where the data for the blockchain is stored. The default home directory is `~/.regen`.
+
+Initialize the node:
+```
+regen init node --chain-id test
+```
+
+In this case, `node` is the name (or "moniker") of the node and `test` is the chain ID. Feel free to change these values but make sure to use the same value for `chain-id` in the following steps.
+
+## Update Genesis
+
+When the node was initialized, a `genesis.json` file was created within the `config` directory. In this section, you will be adding two genesis accounts (accounts with an initial token balance) and a genesis transaction (a transaction that registers the validator account in the validator set).
+
+Add `validator` account to `genesis.json`:
+```
+regen add-genesis-account validator 5000000000stake --keyring-backend test
```

Comment:
can we use different denom than `stake`? may be `uregen`? We can update genesis to use custom denom after `init` using the following command:
```sh
sed -i "s/stake/$DENOM/g" $DAEMON_HOME/config/genesis.json
```